components/SearchField/index.js

```diff
@@ -0,0 +1,141 @@
+import { useState, useEffect } from "react";
+import styled from "styled-components";
+import { AiOutlineSearch, AiOutlineClose } from "react-icons/ai";
+import { keyframes } from "styled-components";
+
+export default function SearchField({ value, handleSearchChange }) {
+  const [showSearch, setShowSearch] = useState(false);
+
+  const handleSearchClick = () => setShowSearch(!showSearch);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setShowSearch(window.innerWidth >= 900);
+    };
+    window.addEventListener("resize", handleResize);
+    handleResize();
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
+  return (
+    <StyledSearchWrapper>
+      {showSearch && (
+        <StyledSearchFieldWrapper className={showSearch ? "show" : ""}>
```

Comment:
Can you make this a property on the StyledSearchFieldWrapper component?

```suggestion
        <StyledSearchFieldWrapper showSearch={showSearch}>
```

See: https://styled-components.com/docs/basics#adapting-based-on-props